server/controllers/manage/placementController.ts

```diff
@@ -0,0 +1,21 @@
+import type { Request, RequestHandler, Response } from 'express'
+
+import { PremisesService } from '../../services'
+import { DateFormats } from '../../utils/dateUtils'
+
+export default class PlacementController {
+  constructor(private readonly premisesService: PremisesService) {}
+
+  show(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const { premisesId, placementId } = req.params
+      const referrer = req.headers.referer
+      const { user } = res.locals
+      const placement = await this.premisesService.getPlacement({ token: req.user.token, premisesId, placementId })
+      const premises = await this.premisesService.find(req.user.token, req.params.premisesId)
```

Comment:
It seems all the premises info the template ever uses is either the premises `id` or the `name`. I believe these are available directly from `placement.premises`, so we can save an API request here.

Reply:
True. It does seem architecturally wrong for the api calls to be so tailored for the precise requirements of the UI but I guess as the information is there... 
It tidies up quite a lot of parameter passing in the tests too.